package.json

```diff
@@ -173,6 +174,7 @@
 		"@types/validator": "^13.7.1",
 		"@types/webpack-env": "^1.18.4",
 		"@types/wordpress__block-editor": "^11.5.8",
+		"@uiw/react-codemirror": "^4.21.21",
```

Comment:
Ideally, we shouldn't be introducing any new dependencies. I'd be fine with a textarea with a monospace font.